velox/expression/CastExpr-inl.h

```diff
@@ -126,6 +126,23 @@ StringView convertToStringView(
   return StringView(startPosition, writePosition - startPosition);
 }
 
+inline bool hasWhitespace(const StringView& value) {
+  if (value.size() > 0) {
+    char first = value.data()[0];
+    if (first == ' ' || first == '\t' || first == '\n' || first == '\r' ||
+        first == '\v' || first == '\f') {
+      return true;
+    }
+    char last = value.data()[value.size() - 1];
+    if (last == ' ' || last == '\t' || last == '\n' || last == '\r' ||
```

Comment:
would you extract a helper function to avoid copy-paste?

Reply:
Yeah, revised this part